### PR TITLE
Aligned the date field

### LINKED ISSUE #1570: Redesign: Date in row_client_charge.xml not in position
**Summary:**

Date in row_client_charge.xml not in correct position

**Steps to reproduce:**

Login into the app and open "Charges"

**Expected behaviour:**

Date should be aligned

**Observed behaviour:**

Date not aligned.

**Device and Android version:**
Samsung J-6. Android 10.

**Screenshots:**
Screenshot_20201124-191325_Mifos Mobile

### PR DESCRIPTION
**Pull request by ritish099**
Fixes #1570

Screenshot_20201125-192022_Mifos Mobile

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by iamsh4shank — APPROVED** [review]
LGTM

File: app/src/main/res/layout/row_client_charge.xml [hunk 1]
```diff
@@ -41,18 +41,16 @@
...
-                    android:id="@+id/tv_due_date"
-                    android:layout_height="wrap_content"
-                    android:layout_width="match_parent"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
-                    tools:text="@string/medium_text"/>
-
+                    tools:text="@string/medium_text" />
             </LinearLayout>
 
+            <TextView
```
Comment (miPlodder) [inline app/src/main/res/layout/row_client_charge.xml:41]: This code works, but we should use layouts properly as good practice.
We can remove parent `LinearLayout`. Please check.

**Review by miPlodder — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/row_client_charge.xml (hunk 1 above)
Comment (ritish099) [reply app/src/main/res/layout/row_client_charge.xml:41]: Ok. Working on it. @miPlodder

**Comment by ritish099** [comment]
@miPlodder Removed the linear layout. Kindly check it out.

**Comment by miPlodder** [comment]
Changes look good to me now, But can we follow https://github.com/openMF/mifos-mobile/wiki/Commit-Style-Guide and stash the commits.

**Comment by ritish099** [comment]
@miPlodder  check it out

**Comment by miPlodder** [comment]
Can we add `commit type` also.
`fix#<issueId>: <commit msg>`

**Comment by ritish099** [comment]
@miPlodder changed the commit message. Is it ok now?

**Comment by miPlodder** [comment]
I meant issue id #1570. You have added pr id.

**Comment by ritish099** [comment]
@miPlodder check.......

**Comment by miPlodder** [comment]
Nice Job, keep contributing :)
Learn from the comments I provided, and ensure you don't do them again :)

**Pull request merged**